docs/BindingWithGrammar.md

```diff
@@ -0,0 +1,131 @@
+# Binding with Grammar
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides support for automatic XML schema/grammar binding through generation and/or using an existing `.xsd/.dtd` file.
+
+[Validation with XSD grammar](Validation.md#validation-with-xsd-grammar) and [Validation with DTD grammar](Validation.md#validation-with-dtd-grammar) provide more information on manual binding of an XML document.
+
+In the following sections, we would like to bind the XML document `foo.xml`:
+
+```xml
+<foo>
+  <bar />
+</foo>
+```
+
+with an associated grammar file.
+
+## Binding with Existing Grammar
+
+Consider an XSD document, `foo.xsd` (in the same directory as `foo.xml`), defined as follows :
+
+```xsd
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="foo">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="bar" />
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+</xs:schema>
+```
+
+and a DTD document, `foo.dtd` (in the same directory as `foo.xml`), defined as follows:
+```dtd
+<!ELEMENT foo (bar)>
+<!ELEMENT bar EMPTY>
+```
+
+Either file can be bound using the [XML Binding Wizard](#the-xml-binding-wizard), which can be triggered by [Command](#command), [CodeLens](#codelens) or [Quick Fix](#quick-fix).
+
+### The XML Binding Wizard
+
+After opening the binding wizard using one of the methods mentioned above, a dropdown with the following 2 options will appear at the Command Palette:
+
+![Binding Wizard Dropdown](./images/BindingWithGrammar/BindingWizardDropdown.png)
+
+1. Standard (xsi, DOCTYPE)
+
+If the selected file is an XSD file, this option will bind the grammar file by adding the `xmlns:xsi` (with the value as "http://www.w3.org/2001/XMLSchema-instance") and `xsi:noNamespaceSchemaLocation` (with the value as the path to the grammar file) attributes to the root tag:
```

Comment:
xsi:schemaLocation is supported two. When you try to bind a xsd which have a namespace, so please add that.